commands/timeout.js

```diff
@@ -24,58 +36,52 @@ module.exports = {
       return;
     }
 
-    for (var [id, user] of message.mentions.users) {
```

Comment:
Also, I just remembered the reason why I removed it, so I could dirty check for the time argument by just shifting off the first 2 :|

I should fix that...